plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -96,19 +98,24 @@ servers = [
 # database_type = "AzureSQLDB"
 
 ## A list of queries to include. If not specified, all the above listed queries are used.
-# include_query = []
+# include_query = ["AzureSQLDBResourceStats", "AzureSQLDBResourceGovernance", "AzureSQLDBWaitStats", "AzureSQLDBDatabaseIO", "AzureSQLDBServerProperties", "AzureSQLDBOsWaitstats", "AzureSQLDBMemoryClerks", "AzureSQLDBPerformanceCounters", "AzureSQLDBRequests", "AzureSQLDBSchedulers"]
```

Comment:
Hey @andrewzayats, you changed this in the documentation, but not in the code. With the current code, your new query will automagically appear with old configs. Please initialize `IncludeQuery` as described here.